DUnitX.Assert.pas

```diff
@@ -359,6 +361,23 @@ class procedure Assert.AreEqual(const expected, actual: boolean; const message:
     FailFmt(SUnexpectedErrorStr ,[BoolToStr(expected, true), BoolToStr(actual, true), message], ReturnAddress);
 end;
 
+class procedure Assert.AreEqual(const expected, actual: string; const compformat: TDUnitXComparableFormatClass; const ignoreCase: boolean; const message: string);
```

Comment:
The AreEqual seems not to be using TDUnitXComparableFormatClass before the test is failed with the creation of the exception object. If it isn't used in the determination of failing the test, is this adding a purpose to the assert class  other than the failure/passing of tests?